crates/common/src/lib.rs

```diff
@@ -14,29 +17,47 @@ use slight_http_api::HttpHandlerData;
 ///
 /// It contains:
 ///     - a `resource_map`,
-///     - a `secret_store`, and
+///     - a `implementor`,
+///     - a `name`,
+///     - a `configs_map`, and
 ///     - the `slightfile_path`.
 #[derive(Clone, Default)]
 pub struct BasicState {
     pub resource_map: ResourceMap,
-    pub secret_store: String,
+    pub implementor: String,
+    pub name: String,
+    pub configs_map: Option<HashMap<String, String>>,
```

Comment:
```suggestion
    pub configs_map: HashMap<String, String>,
```
because HashMap has a empty default.

Reply:
discussed in chat ✅